src/libraries/Microsoft.PowerFx.Interpreter/Functions/StandardErrorHandling.cs

```diff
@@ -3,7 +3,9 @@
 
 using System;
 using System.Collections.Generic;
+using System.Data;
 using System.Linq;
+using System.Runtime.InteropServices;
```

Comment:
Do we need those new using statements?

Reply:
Weird, idk why vs added them, removing it...